source/classes.tex

```diff
@@ -173,12 +173,14 @@
 
 $M(\mathtt{X})$ is defined as follows:
 \begin{itemize}
-\item If \tcode{X} is a non-union class type, the set $M(\mathtt{X})$ is
-empty if \tcode{X} has no (possibly inherited (Clause~\ref{class.derived}))
-non-static data members;
-otherwise, it consists of the type of the first non-static data member
-of \tcode{X} (where said member may be an anonymous union), $\mathtt{X}_0$,
-and the elements of $M(\mathtt{X}_0)$.
+\item If \tcode{X} is a non-union class type with no (possibly
+inherited (Clause~\ref{class.derived})) non-static data members, the set
+$M(\mathtt{X})$ is empty.
+
+\item If \tcode{X} is a non-union class type whose first non-static data
+member has type $\mathtt{X}_0$ (where said member may be an anonymous union),
```

Comment:
Everywhere else where you've been making changes you used a style like `\tcode{X}$_0$` -- any reason to prefer one over the other? (Personally, I prefer this, one, though I would perhaps make a macro `\mcode` for this use.)